### ISSUE TITLE
Readme missing instruction about PrimaryAuthorSlug

### ISSUE THREAD

**Issue by 1hitsong** [hades200082/ButterCMS-dotNetcore-BoilerPlate#1]
I downloaded the .net core boilerplate, changed the JSON settings to my API token as instructed. I click run in visual studio and I receive an error from shared/layout.cshtml that Object reference not set to an instance of an object on line 62.

**Comment by jakelumetta** [comment]
Another person reported this issue:  ` I was then getting nullrefExceptions related to the view bag author on line 62 of the shared layout on this line @if (!string.IsNullOrEmpty((((Author) ViewBag.Author).ProfileImage)))`

**Comment by hades200082** [comment]
@1hitsong Thanks for the report and my apologies for taking so long to get back to you. My github account seemed to stop sending me notifications and I've had no reason to look at this repo since I finished building my site.

Thanks, @jakelumetta, for giving me a nudge.

---

It seems that I missed a step out of the instructions. You also need to set the `PrimaryAuthorSlug` in the appsettings.

This is usually defined as `<firstname>-<lastname>` from the relevant person's ButterCMS user account. It's also a good idea for the relevant user account to have a non-null "Profile Image URL" too since this is used within the layout.

Obviously, this being a boilerplate, I'd expect that you'd want to change the layout anyway.

I've updated the readme accordingly.

@jakelumetta - When I first built this I'm sure I found my author-slug in the CMS somewhere ... has the UI changed?
